release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Release notes
 
+## March 7, 2019
+
+The following update was made to the Mobile Core extension in Android version 1.1.1:
+
+* Bug Fix - Resolved an issue that could result in a crash when the underlying analytics database was corrupted.
```

Comment:
- Please remove the text "Bug Fix".
- Please capitalize "analytics".

Reply:
done